### ISSUE TITLE
UnusedPrivateMember rule in abstract functions

### ISSUE THREAD

**Issue by PHELAT** [arturbosch/detekt#983]
## Expected Behavior
abstract function parameters aren't private members and i can't do anything to fix this error.

## Current Behavior
I'm getting this error from detekt
```
UnusedPrivateMember Private member is unused.
BaseViewHolder.kt:13:23
Function parameter widgetType is unused
```

## Steps to Reproduce (for bugs)
Just copy paste my class into your project
Also make sure that you are using `1.0.0-RC7-3` version of detekt, you can't reproduce it with older versions like `RC7-2`
Use this rule :
```yml
  UnusedPrivateMember:
    active: true
    allowedNames: "(_|ignored|expected)"
```

## Context
I have this abstract class in my project 
```kotlin
abstract class BaseViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    abstract fun bind(widgetType: BaseWidgetEntity)

}
```

## Your Environment
* Version used: 1.0.0-RC7-3
* Operating System and version: macOS High Sierra 10.13.1

**Comment by marianosimone** [comment]
This seems like a duplicate of #972 (already fixed in master... we'll just need to wait for the next version to be released)
